src/treeview/treeview.scss

```diff
@@ -18,9 +18,31 @@
       list-style: none;
       cursor: pointer; }
   
-  .rstcustom__collapseButton, .rstcustom__expandButton {
-    display: none; }
   
-  .glyphicon {
-    display: flex !important; }
-  
\ No newline at end of file
+.tree {
```

Comment:
we should avoid generic class names like this "tree" to avoid the possibility for overrides.